internal/function/uast.go

```diff
@@ -206,7 +207,8 @@ func (u *uastFunc) getUAST(
 				return nil, nil
 			}
 
-			return nil, err
+			logrus.WithField("err", err).Error("unable to get UAST from bblfsh")
+			return nil, nil
```

Comment:
if we are not going to return an error,  the previous `if ErrParseBlob.Is(err)` is not necessary anymore

Reply:
It's there to not pollute the logs with infinite "can't parse"s

Reply:
Then, what should be inside an `if` is the logging line maybe.